scrapscript.py

```diff
@@ -1285,6 +1336,12 @@ def test_tokenize_multiple_digits(self) -> None:
     def test_tokenize_negative_int(self) -> None:
         self.assertEqual(tokenize("-123"), [Operator("-"), IntLit(123)])
 
+    def test_tokenize_float(self) -> None:
+        self.assertEqual(tokenize("3.14"), [FloatLit(3.14)])
+
+    def test_tokenize_negative_float(self) -> None:
+        self.assertEqual(tokenize("-3.14"), [Operator("-"), FloatLit(3.14)])
+
```

Comment:
Add tests for `1.`, `.1`, `1.0.2`, `1..`, etc

Reply:
Good call, thanks